Solutions/AristaAwakeSecurity/Analytic Rules/HighMatchCountsByDevice.yaml

```diff
@@ -0,0 +1,64 @@
+id: 90b7ac11-dd6c-4ba1-a99b-737061873859
+name: Awake Security - High Match Counts By Device
+description: This query searches for devices with unexpectedly large number of activity match.
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AristaAwakeSecurity
+    dataTypes:
+      - AwakeSecurity
```

Comment:
If the detection or hunting query operates on a Kusto Function/ Parser instead of the table (like Syslog, CommonEventFormat, _CL), dataTypes should be the Kusto Function name / Parser name and not the table name. However, looking at the query it seems that we are using just the table CommonSecurityLog and not some parser. Please correct me if I am mistaken here. If we are just using the table the dataTypes value should generally be the tablename i.e : CommonSecurityLog instead of AwakeSecurity